Emby.Dlna/PlayTo/PlayToController.cs

```diff
@@ -736,6 +764,10 @@ private async Task SetAudioStreamIndex(int? newIndex)
 
                     await _device.SetAvTransport(newItem.StreamUrl, GetDlnaHeaders(newItem), newItem.Didl, CancellationToken.None).ConfigureAwait(false);
 
+                    // Send a message to the DLNA device to notify what is the next track in the play list.
+                    var newItemIndex = _playlist.FindIndex(item => item.StreamUrl == newItem.StreamUrl);
```

Comment:
Isn't SeekAfterTransportChange unnecessary if SendNextTrackMessage is supported?

Reply:
Also, do you know if dlna device, when the track changes, still sends a change notification?

Reply:
I don't know. In the code here, there is a timer that periodicaly polls the DLNA device status and synchronizes with it. That suggests there isn't any kind of callback notification (at the time this code was written).

Reply:
Sorry - confusing this with the newer version which implements dlna callbacks. (The device triggers updates on changes.)